### PR TITLE
[TPCDS][TEST] Fix TPCDS suite issue and introduce tpcds queries(ten plus) 

### PR DESCRIPTION
**Pull request by turboFei**
### _Why are the changes needed?_

Fix tpcds schema issue and covert all tests, whose file is named like q14a.sql.

Introduce tpcds queries, whose index between ten and twenty.

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala [hunk 1]
```diff
@@ -185,6 +185,7 @@ trait TPCDSHelper {
       Field("cr_ship_mode_sk", int),
       Field("cr_warehouse_sk", int),
       Field("cr_reason_sk", int),
+      Field("cr_order_number", int),
```
Comment (turboFei) [inline kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:188]: missed column

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSOutputSchemaSuite.scala [hunk 2]
```diff
@@ -94,22 +94,21 @@ class TPCDSOutputSchemaSuite extends WithKyuubiServer with JDBCTestUtils with TP
...
+        q.toFile.listFiles().filter(_.getName.endsWith(".sql")).foreach { qf =>
+          val schemaFile = Paths.get(
+            baseResourcePath.toFile.getAbsolutePath,
+            name,
+            q.getFileName.toString,
+            s"${qf.getName.stripSuffix(".sql")}.output.schema"
+          )
+          val queryString = fileToString(qf.toPath)
+          runQuery(queryString, schemaFile)
+        }
```
Comment (turboFei) [inline kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSOutputSchemaSuite.scala:111]: cover sql files named q14a.sql

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala [hunk 3]
```diff
@@ -475,6 +476,7 @@ trait TPCDSHelper {
       Field("ca_county", varchar30),
       Field("ca_state", char2),
       Field("ca_zip", char10),
+      Field("ca_country", char20),
```
Comment (turboFei) [inline kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:479]: ddito

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala (hunk 1 above)
Comment (pan3793) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:188]: Is the schemas different in different TPCDS versions? Or we just missed the column.

Comment (turboFei) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSHelper.scala:188]: just missed

**Review by pan3793 — APPROVED** [review]
LGTM

**Comment by pan3793** [comment]
Thanks, merged to master for v1.2.0

File: kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSOutputSchemaSuite.scala (hunk 2 above)
Comment (ulysses-you) [reply kyuubi-main/src/test/scala/org/apache/kyuubi/operation/tpcds/TPCDSOutputSchemaSuite.scala:111]: We can make directory for every sql even them belong to the same query index. But this change is also fine.

**Pull request closed without merging**